lib/auth/index.tsx

```diff
@@ -60,14 +62,58 @@ export class Auth extends Component<Props> {
     const helpMessage = isCreatingAccount
       ? 'Already have an account?'
       : "Don't have an account?";
-    const errorMessage = isCreatingAccount
-      ? 'Could not create account. Please try again.'
-      : 'Could not log in with the provided email address and password.';
+
+    const errorMessage = isCreatingAccount ? (
+      <>
```

Comment:
This doesn't really matter, but for my own curiosity. I've seen in this PR both `<>` and `<Fragment>` used. Does that matter, or is there a preference one over the other?

Reply:
I learned about `<>` like last week. I don't know which we prefer but I guess I can make it consistent..

I think it's the case that `<>` doesn't require importing `Fragment`? So maybe that's better

Reply:
We should decide what to use and stick with it. I may have used `<>` since I was fast coding and not really thinking about where I was and code style. For now, let's change it to use Fragment since that is the code standard. It all compiles to the same thing so it is whatever feels right to yous.